stubs/setuptools/setuptools/_distutils/command/bdist.pyi

```diff
@@ -0,0 +1,25 @@
+from _typeshed import Unused
+from typing import ClassVar
+from typing_extensions import deprecated
+
+from ..cmd import Command
+
+def show_formats() -> None: ...
+
+class ListCompat(dict[str, tuple[str, str]]):
+    @deprecated("format_commands is now a dict. append is deprecated")
+    def append(self, item: Unused) -> None: ...
+
+class bdist(Command):
+    description: ClassVar[str]
+    user_options: ClassVar[list[tuple[str | None, ...]]]
```

Comment:
Just eyeballing the runtime code, it looks like the tuples always have three elements?

```suggestion
    user_options: ClassVar[list[tuple[str | None, str | None, str | None]]]
```